### PR TITLE
James 1850-Modify JSPF top bar links

### PR DESCRIPTION
**Pull request by chibenwa**
We had compilation issues with rat-plugin. We copied configuration from james-project.

### CODE REVIEW

**Comment by Laura-Royet** [comment]
Could you add a screen shot of the page please?

**Comment by chibenwa** [comment]
We just changes a menu, is it really needed ?

**Comment by Laura-Royet** [comment]
Yes I think it is useful to check the result.

**Comment by chibenwa** [comment]
screenshot from 2016-10-28 15-15-54

**Review by Laura-Royet — APPROVED** [review]

File: src/site/site.xml [hunk 1]
```diff
@@ -40,45 +40,30 @@
...
-                    href="http://james.apache.org/protocols/index.html" />
-            <item
-                    name="Mailets"
-                    href="http://james.apache.org/mailet/index.html" />
+
+        <links>
             <item
-                    name="Mailbox"
-                    href="http://james.apache.org/mailbox/index.html" />
+                name="Home"
```
Comment (aduprat) [inline src/site/site.xml]: What is the difference between `Home` & `James` (same link)

Comment (aduprat) [inline src/site/site.xml:46]: What is the difference between `Home` & `James` (same link)

Comment (Laura-Royet) [reply src/site/site.xml:46]: Home is the button leading to James Marketing website and James leads to all the internal james project

**Pull request merged**